packages/blade/src/components/Tooltip/TooltipArrowNative.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+import type { Side } from '@floating-ui/react';
+import * as React from 'react';
+import styled from 'styled-components/native';
+import type { UseFloatingReturn } from '@floating-ui/react-native';
+import { View } from 'react-native';
+import { ARROW_HEIGHT, ARROW_WIDTH } from './constants';
+import Svg, { Path } from '~components/Icons/_Svg';
+import { useTheme } from '~components/BladeProvider';
+import type { SvgProps } from '~components/Icons/_Svg/Svg/types';
+
+type TooltipArrowProps = {
+  context: UseFloatingReturn;
+};
+
+const StyledSvg = styled(Svg)(({ styles }) => {
+  return styles;
+});
+
+const TooltipArrow = React.forwardRef<SvgProps, TooltipArrowProps>(
+  ({ context }, ref): React.ReactElement => {
+    const { theme } = useTheme();
+
+    const {
+      placement,
+      elements: { floating },
+      middlewareData: { arrow },
+    } = context;
+    const width = ARROW_WIDTH;
+    const height = ARROW_HEIGHT;
+    const strokeWidth = theme.border.width.thin * 2;
+
+    if (!ref) {
+      console.warn('Floating UI: The `ref` prop is required for the `FloatingArrow`', 'component.');
+    }
+    if (!floating) {
+      return <></>;
+    }
+
+    const [side] = placement.split('-') as [Side];
```

Comment:
Minor: maybe refactor this logic, being used at other places as well

Reply:
Done.